### PR TITLE
[zh-cn] Localize blog 2022-02-07-sig-multicluster-spotlight

### PR DESCRIPTION
**Pull request by jzhupup**
Added the zh-cn text to this blog:

`content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md`

### CODE REVIEW

File: content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+title: "Spotlight on SIG Multicluster"
+date: 2022-02-07
+slug: sig-multicluster-spotlight-2022
+canonicalUrl: https://www.kubernetes.dev/blog/2022/02/04/sig-multicluster-spotlight-2022/
+-->
+
+<!--
+**Authors:** Dewan Ahmed (Aiven) and Chris Short (AWS) 
+-->
+**作者：** Dewan Ahmed (Aiven) and Chris Short (AWS)
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md:19]:
```suggestion
**作者：** Dewan Ahmed (Aiven) 和 Chris Short (AWS)
```

File: content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+<!-- 
+## Introduction
+
+[SIG Multicluster](https://github.com/kubernetes/community/tree/master/sig-multicluster) is the SIG focused on how Kubernetes concepts are expanded and used beyond the cluster boundary. Historically, Kubernetes resources only interacted within that boundary - KRU or Kubernetes Resource Universe (not an actual Kubernetes concept). Kubernetes clusters, even now, don't really know anything about themselves or, about other clusters. Absence of cluster identifiers is a case in point. With the growing adoption of multicloud and multicluster deployments, the work SIG Multicluster doing is gaining a lot of attention. In this blog, [Jeremy Olmsted-Thompson, Google](https://twitter.com/jeremyot) and [Chris Short, AWS](https://twitter.com/ChrisShort) discuss the interesting problems SIG Multicluster is solving and how you can get involved. Their initials **JOT** and **CS** will be used for brevity. 
+-->
+## 简介
+
+[SIG Multicluster](https://github.com/kubernetes/community/tree/master/sig-multicluster) 专注于 Kubernetes 概念如何在扩展和使用超越集群边界的 SIG.
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md:28]:
```suggestion
[SIG Multicluster](https://github.com/kubernetes/community/tree/master/sig-multicluster) 是专注于 Kubernetes 概念如何拓展并用于集群边界之外的 SIG。
```

File: content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+历史上，Kubernetes 资源仅在这个边界内进行互动 - KRU 或 Kubernetes Resource Universe（不是一个实际的Kubernetes概念）。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md:29]:
```suggestion
以往 Kubernetes 资源仅在 Kubernetes Resource Universe (KRU) 这个边界内进行交互，其中 KRU不是一个实际的 Kubernetes 概念。
```

File: content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+答案几乎总是“视情况而定”。你为什么要这样做？是因为某种合规性使你关注位置吗？是性能问题吗？是可用性吗？
+
+我认为，在我们有了集群 ID 之后，重新审视注册表模式可能是一个自然的步骤，也就是说，你如何将这些集群真正关联在一起？也许你有一个分布式部署，你在世界各地的数据中心运行。
+我想随着更多多集群功能的发展，扩展该领域的 API 将变得很重要。这实际上取决于社区开始使用这些工具做什么。
+
+<!-- 
+**CS**: In the early days of Kubernetes, we used to have a few large Kubernetes clusters and now we're dealing with many small Kubernetes clusters - even multiple clusters for our own dev environments. How has this shift from a few large clusters to many small clusters affected the SIG? Has it accelerated the work or make it challenging in any way? 
+-->
+**CS**：在 Kubernetes 的早期，我们曾经有几个大型的 Kubernetes 集群，而现在我们面对的是许多小型 Kubernetes 集群 - 甚至在我们自己的开发环境中使用多个集群。
+这种从几个大集群到许多小集群的转变对 SIG 有何影响？它是否加快了工作进度或使其具有任何挑战性？
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md:115]:
```suggestion
这种从几个大集群到许多小集群的转变对 SIG 有何影响？它是否加快了工作进度或者在某种程度上使得问题变得更困难？
```

File: content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+
+<!-- 
+**JOT**: I think that it has created a lot of ambiguity that needs solving. Originally, you'd have a dev cluster, a staging cluster, and a prod cluster. When the multi region thing came in, we started needing  dev/staging/prod clusters, per region. And then, sometimes clusters really need more isolation due to compliance or some regulations issues. Thus, we're ending up with a lot of clusters. I think figuring out the right balance on how many clusters should you actually have is important. The power of Kubernetes is being able to deploy a lot of things managed by a single control plane. So, it's not like every single workload that gets deployed should be in its own cluster. But I think it's pretty clear that we can't put every single workload in a single cluster. 
+-->
+**JOT**：我认为它造成了很多需要解决的歧义。最初，您将拥有一个 dev 集群、一个 staging 集群和一个 prod 集群。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md:120]:
```suggestion
**JOT**：我认为它带来了很多需要解决的歧义。最初，你可能拥有一个 dev 集群、一个 staging 集群和一个 prod 集群。
```

File: content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+当多区域的事情出现时，我们开始需要每个区域的 dev/staging/prod 集群。然后，有时由于合规性或某些法规问题，集群确实需要更多的隔离。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md:121]:
```suggestion
当引入了多区域的考量时，我们开始需要在每个区域部署 dev/staging/prod 集群。
再后来，有时由于合规性或某些法规问题，集群之间确实需要更强的隔离。
```

File: content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+因此，我们最终会有很多集群。我认为在你究竟应该有多少个集群上找到平衡是很重要的。Kubernetes 的强大之处在于能够部署由单个控制平面管理的大量事物。
+因此，并不是每个被部署的工作负载都应该在自己的集群中。但我认为很明显，我们不能将每一个工作负载都放在一个集群中。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md:123]:
```suggestion
因此，并不是每个被部署的工作负载都应该在自己的集群中。
但是，我认为同样很明显的是，我们不能将所有工作负载都放在一个集群中。
```

File: content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+
+<!-- 
+**CS**: What are some of your favorite things about this SIG?
+
+**JOT**: The complexity of the problems, the people and the newness of the space. We don't have right answers and we have to figure this out. At the beginning, we couldn't even think about multi clusters because there was no way to connect services across clusters. Now there is and we're starting to go tackle those problems, I think that this is a really fun place to be in because I expect that the SIG is going to get a lot busier the next couple of years. It's a very collaborative group and we definitely would like more people to come join us, get involved, raise their problems and bring their ideas.  
+-->
+**CS**：你最喜欢 SIG 的哪些方面？
+
+**JOT**：问题的复杂性、人员和空间的新颖性。我们没有正确的答案，我们必须弄清楚这一点。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md:132]:
```suggestion
**JOT**：问题的复杂性、人的因素和问题空间的新颖性。我们还没有正确的答案，而我们必须找到正确的答案。
```

File: content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+一开始，我们甚至无法考虑多集群，因为无法跨集群连接服务。
+现在我们开始着手解决这些问题，我认为这是一个非常有趣的地方，因为我预计 SIG 在未来几年会变得更加繁忙。
+这是一个非常协作的团体，我们绝对希望更多的人参与并加入我们，提出他们的问题和想法。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md:135]:
```suggestion
这是一个协作很密切的团体，我们绝对希望更多的人参与、加入我们，提出他们的问题和想法。
```

File: content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md [hunk 10]
```diff
@@ -0,0 +1,159 @@
...
+
+<!-- 
+**CS**: What do you think keeps people in this group? How has the pandemic affected you? 
+-->
+**CS**：你认为是什么让人们留在这个群体中？疫情对您有何影响？
+
+<!-- 
+**JOT**: I think it definitely got a little bit quieter during the pandemic. But for the most part; it's a very distributed group so whether you're calling in to our weekly meetings from a conference room or from your home, it doesn't make that huge of a difference. During the pandemic, a lot of people had time to focus on what's next for their scale and growth. I think that's what keeps people in the group - we have real problems that need to be solved which are very new in this space. And it's fun :) 
+-->
+**JOT**：我认为在疫情期间它肯定会变得安静一些。但在大多数情况下，这是一个非常分散的小组，因此无论您在会议室或者在家中参加我们的每周会议，都不会产生太大的影响。在疫情期间，很多人有时间专注于他们接下来的规模和增长。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md:145]:
```suggestion
**JOT**：我认为在疫情期间这个群体肯定会变得安静一些。但在大多数情况下，这是一个非常分散的小组，
因此无论您在会议室或者在家中参加我们的每周会议，都不会产生太大的影响。
在疫情期间，很多人有时间专注于他们接下来的规模和增长。
```

File: content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md [hunk 11]
```diff
@@ -0,0 +1,159 @@
...
+
+**CS**: That's all we have for today. Thanks Jeremy for your time.
+
+**JOT**: Thanks Chris. Everybody is welcome at our [bi-weekly meetings](https://github.com/kubernetes/community/tree/master/sig-multicluster#meetings). We love as many people to come as possible and welcome all questions and all ideas. It's a new space and it'd be great to grow the community. 
+-->
+## 结束语
+
+**CS**：这就是我们今天的全部内容，感谢 Jeremy 的时间。
+
+**JOT**：谢谢 Chris。我们的 [bi-weekly meetings](https://github.com/kubernetes/community/tree/master/sig-multicluster#meetings) 欢迎所有人参加。我们希望尽可能多的人前来，并欢迎所有问题与想法。这是一个新的空间，如果能让社区发展起来，那就太好了。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md:159]:
```suggestion
**JOT**：谢谢 Chris。我们的 [bi-weekly meetings](https://github.com/kubernetes/community/tree/master/sig-multicluster#meetings)
欢迎所有人参加。我们希望尽可能多的人前来，并欢迎所有问题与想法。
这是一个新的领域，如果能让社区发展起来，那就太好了。
```

File: content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md (hunk 8 above)
Comment (jzhupup) [reply content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md]: 应该是人的因素和领域的新颖性

File: content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md [hunk 12]
```diff
@@ -0,0 +1,177 @@
...
+为了满足这一特定需求，我们不再考虑一个个 Kubernetes 集群，而是考虑 ClusterSets — 在某种范围内协同工作的集群分组。
+与考虑所有时间点和所有空间位置上存在的集群相比，这一范畴要窄得多。这一概念还为实现者提供了定义边界（ClusterSet）的灵活性，在该边界之外，该集群 ID 将不再是唯一的。
+
+<!-- 
+**CS**: How do you feel about the current state of SIG Multicluster versus where you're hoping to be in future?
+
+**JOT**: There's a few projects that are kind of getting started, for example, Work API. In the future, I think that some common practices around how do we deploy things across clusters are going to develop. 
+> If I have clusters deployed in a bunch of different regions; what's the best way to actually do that? 
+-->
+**CS**：您对 SIG Multicluster 的现状有何看法，对于您希望未来的达到的目标有何想法？
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md:107]:
```suggestion
**CS**：你对 SIG Multicluster 的现状有何看法，对于你希望未来的达到的目标有何想法？
```

File: content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md [hunk 13]
```diff
@@ -0,0 +1,177 @@
...
+
+**JOT**：问题的复杂性、人的因素和领域的新颖性。我们还没有正确的答案，我们必须找到正确的答案。
+一开始，我们甚至无法考虑多集群，因为无法跨集群连接服务。
+现在我们开始着手解决这些问题，我认为这是一个非常有趣的地方，因为我预计 SIG 在未来几年会变得更加繁忙。
+这是一个协作很密切的团体，我们绝对希望更多的人参与、加入我们，提出他们的问题和想法。
+
+<!-- 
+**CS**: What do you think keeps people in this group? How has the pandemic affected you? 
+-->
+**CS**：你认为是什么让人们留在这个群体中？疫情对您有何影响？
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md:155]:
```suggestion
**CS**：你认为是什么让人们留在这个群体中？疫情对你有何影响？
```

File: content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md [hunk 14]
```diff
@@ -0,0 +1,177 @@
...
+
+<!-- 
+**JOT**: I think it definitely got a little bit quieter during the pandemic. But for the most part; it's a very distributed group so whether you're calling in to our weekly meetings from a conference room or from your home, it doesn't make that huge of a difference. During the pandemic, a lot of people had time to focus on what's next for their scale and growth. I think that's what keeps people in the group - we have real problems that need to be solved which are very new in this space. And it's fun :) 
+-->
+**JOT**：我认为在疫情期间这个群体肯定会变得安静一些。但在大多数情况下，这是一个非常分散的小组，
+因此无论您在会议室或者在家中参加我们的每周会议，都不会产生太大的影响。在疫情期间，很多人有时间专注于他们接下来的规模和增长。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md:161]:
```suggestion
因此无论你在会议室或者在家中参加我们的每周会议，都不会产生太大的影响。在疫情期间，很多人有时间专注于他们接下来的规模和增长。
```

File: content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md [hunk 15]
```diff
@@ -0,0 +1,177 @@
...
+
+**CS**: That's all we have for today. Thanks Jeremy for your time.
+
+**JOT**: Thanks Chris. Everybody is welcome at our [bi-weekly meetings](https://github.com/kubernetes/community/tree/master/sig-multicluster#meetings). We love as many people to come as possible and welcome all questions and all ideas. It's a new space and it'd be great to grow the community. 
+-->
+## 结束语
+
+**CS**：这就是我们今天的全部内容，感谢 Jeremy 的时间。
+
+**JOT**：谢谢 Chris。我们的 [bi-weekly meetings](https://github.com/kubernetes/community/tree/master/sig-multicluster#meetings)
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md:175]:
```suggestion
**JOT**：谢谢 Chris。我们的[雙週會議](https://github.com/kubernetes/community/tree/master/sig-multicluster#meetings)
```

File: content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md [hunk 16]
```diff
@@ -0,0 +1,177 @@
...
+-->
+## 谈话总结
+
+**CS**：SIG Multicluster 存在多久了？SIG 在起步阶段情况如何？ 你参与这个 SIG 多长时间了？
+
+**JOT**：我在 SIG Multicluster 工作了将近两年。 我所知道的关于初创时期的情况都来自传说，但即使在早期，也一直是为了解决相同的问题。
+早期工作的例子之一是 [KubeFed](https://github.com/kubernetes-sigs/kubefed)。
+我认为仍然有一些人在使用 KubeFed，但它只是一小部分。
+那时，我认为人们在部署大量 Kubernetes 集群时，还没有达到我们拥有大量实际具体用例的地步。
+像 KubeFed 和 [Cluster Registry] (https://github.com/kubernetes-retired/cluster-registry)
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-02-07-sig-multicluster-spotlight/index.md:52]:
```suggestion
像 KubeFed 和 [Cluster Registry](https://github.com/kubernetes-retired/cluster-registry)
```

**Review by windsonsea — COMMENTED** [review]
Preview FYI:
https://deploy-preview-36146--kubernetes-io-main-staging.netlify.app/zh-cn/blog/2022/02/07/sig-multicluster-spotlight-2022/

**Comment by Sea-n** [comment]
Hey @jzhupup,

It's been a while since the last update, would you like to address issues in the previous comments so that we can move forward?

**Comment by jzhupup** [comment]
@Sea-n se  I'm so sorry I missed it. Now, I have processed all the comments

**Pull request merged**